main/main.ino

```diff
@@ -1319,6 +1319,10 @@ void loop() {
       PilighttoMQTT();
 #endif
 #ifdef ZgatewayBT
+#  ifdef ZmqttDiscovery
+      if (disc)
+        launchBTDiscovery();
+#  endif
```

Comment:
With the discovery here, we are going to launch it without any delay or wait time